### PR TITLE
Feature/new node object selection

### PR DESCRIPTION
**Pull request by Fenrir12**
Corrected nodes for better functionality.
Added subscibre to topic of object to find
Added publish of position of selected objct
Wrote launch file to test along with ork line mod server and client

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+#add_message_files(
+#   FILES
+#   Message1.msg
+#   Message2.msg
+# )
+
+## Generate services in the 'srv' folder
+add_service_files(
+  FILES
+  rcgnzd_obj.srv
```
Comment (maximeconnolly) [inline CMakeLists.txt]: Change to RecognizeObject.srv

File: srv/rcgnzd_obj.srv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+object_recognition_msgs/RecognizedObjectArray objectarray
+sensor_msgs/PointCloud2 pcl_ref
+string filter
+---
+geometry_msgs/PoseWithCovarianceStamped SlctObjectPose
```
Comment (maximeconnolly) [inline srv/rcgnzd_obj.srv]: change to selected_object_pose

**Review by maximeconnolly — CHANGES_REQUESTED** [review]

**Comment by Fenrir12** [comment]
Note : Error on ServerResponse. Always sends back the first element of the list and not the right element

File: source/wm_obj_slct_srv.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    for i in objects_info_list:
+        if i.name == req.filter:
+            print "Object found: The object with id",req.objectarray.objects[position].type.key, "is :",i.name, "\n\rat position\r\n", req.objectarray.objects[position].pose.pose.pose.position
+            break
+        else:
+            rospy.logout("Object not found, try again")
+        position = position +1
+    rospy.logout("Service select_object_server waiting")
+
+    return req.objectarray.objects[0].pose
```
Comment (maximeconnolly) [inline source/wm_obj_slct_srv.py]: There is the problem the index value of the array is hardcoded.

**Comment by Fenrir12** [comment]
Did the modifications and corrected the retrieved object bug. Latest commit 11013a3 ready to merge

**Review by maximeconnolly — APPROVED** [review]

**Pull request merged**